airflow/providers/amazon/aws/operators/emr.py

```diff
@@ -1025,11 +1031,12 @@ def __init__(
         self.aws_conn_id = aws_conn_id
         self.release_label = release_label
         self.job_type = job_type
-        self.wait_for_completion = wait_for_completion
+        self.wait_for_completion = False if deferrable else wait_for_completion
```

Comment:
this is not really needed, is it ?
I was actually wondering with the new default if it'd make sense to defer only if `deferrable && wait_for_completion` ?